programs/vote/src/vote_instruction.rs

```diff
@@ -103,6 +103,20 @@ pub enum VoteInstruction {
     ///   1. `[SIGNER]` Vote authority
     UpdateVoteStateSwitch(VoteStateUpdate, Hash),
 
+    /// Update the onchain vote state for the signer.
+    ///
+    /// # Account references
+    ///   0. `[Write]` Vote account to vote with
+    ///   1. `[SIGNER]` Vote authority
+    CompactUpdateVoteState(CompactVoteStateUpdate),
+
+    /// Update the onchain vote state for the signer along with a switching proof.
+    ///
+    /// # Account references
+    ///   0. `[Write]` Vote account to vote with
+    ///   1. `[SIGNER]` Vote authority
+    CompactUpdateVoteStateSwitch(CompactVoteStateUpdate, Hash),
+
```

Comment:
@AshwinSekar Please only add new enum variants at the very end.
Adding new enum variants in the middle is not backward compatible:
https://discord.com/channels/428295358100013066/439194979856809985/1006620618101370931

@carllin Please do watch for this when you are reviewing code.